docs/home/contribute/participating.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Participating in SIG-DOCS
+---
+
+{% capture overview %}
+
+SIG-DOCS is one of the [special interest groups](https://github.com/kubernetes/community/blob/master/sig-list.md) within the Kubernetes project, focused on writing, updating, and maintaining the documentation for Kubernetes as a whole.
+
+{% endcapture %}
+
+{% capture body %}
+
+SIG Docs welcomes content and reviews from all contributors. Anyone can open a pull request (PR), and anyone is welcome to comment on content or pull requests in progress.
+
+Within the Kubernetes project, you may also become a member, reviewer, or approver.
+These roles confer additional privileges and responsibilities when it comes to approving and committing changes.
+See [community-membership](https://github.com/kubernetes/community/blob/master/community-membership.md) for more information on how membership works within the Kubernetes community.
+
+## Roles and Responsibilities
+
+- Members
+
+Any member of the [Kubernetes organization](https://github.com/kubernetes) can review a pull request, and SIG Docs team members frequently request reviews from members of other SIGs for technical accuracy.
+SIG Docs also welcomes reviews and feedback regardless of Kubernetes org membership.
+You can indicate your approval by adding a comment of `/lgtm` to a pull request.
+
+- Reviewers
+
+Reviewers are individuals who review documentation pull requests. 
+
+Our automation assigns reviewers to pull requests, and contributors can request your review with a comment on the pull request: `/assign [your_github_handle]`.
+To indicate that a pull request requires no further changes, a reviewer should add comment to the pull request `/lgtm`.
+
+Approvers can indicate their approval with a comment to the pull request: `/approve`.
+
+Reviewers and approvers can add a `/hold` comment to prevent the pull request from being merged. Another reviewer or approver can remove a hold with the comment: `/hold cancel`.
+
+The automation reads the /hold, /lgtm, and /approve comments and sets labels on the pull request.
+When both the lgtm and approve labels exist, and there are no hold labels, the automation on the repository merges the pull request.
+
+**Note:** There is a special case when an approver uses the comment: `/lgtm`. In these cases, the automation will add both `lgtm` and `approve` tags, skipping any further review.
+{: .note }
+
+When a reviewer is assigned a pull request to review it is not a sole responsibility, and any other reviewer may also offer their opinions on the pull request.
+If a reviewer is requested, it is generally expected that the PR will be left to that reviewer to do their editorial pass on the content.
```

Comment:
If a PR author or SIG Docs maintainer requests a review, refrain from merging or closing the PR until the requested reviewer completes their review.